Sources/UtiliKit/General/UIView+Extensions.swift

```diff
@@ -105,4 +105,23 @@ public extension UIView {
         superview.addConstraints([centerXAnchor.constraint(equalTo: superview.centerXAnchor),
                                   centerYAnchor.constraint(equalTo: superview.centerYAnchor)])
     }
+    
+    /// Constrains the receiver to the size of its superview. Optionaly width and height scales can be specified, otherwise they will default to 1.0 (no scaling).
+    func constrainSizeToSuperview(widthScale: CGFloat = 1.0, heightScale: CGFloat = 1.0) {
+        guard let superview = superview else { return }
+        
+        superview.addConstraints([widthAnchor.constraint(equalTo: superview.widthAnchor, multiplier: widthScale),
```

Comment:
The documentation doesn't indicate that it's deprecated, but it does say 
> When developing for iOS 8.0 or later, use the NSLayoutConstraint class’s activate(_:) method instead of calling the addConstraints(_:) method directly. The activate(_:) method automatically adds the constraints to the correct views.
So I guess `NSLayoutConstraint.activate([])` takes out the need to know the correct view to add the constraints to. I was using `view.addConstraints` to match the style of the other functions in this extension- should I update all of them to use `NSLayoutConstraint.activate([])`?

Reply:
Don't worry about updating the rest of the `view.addConstraints` right now. I created #97 to track that work. 😄

Reply:
Should I push this one as it, then pick up #97 to do all the `addConstraints`?

Reply:
Yeah, that works! 👍